tests/Behat/Context/FixtureContext.php

```diff
@@ -44,6 +46,31 @@ public function theHasAContentElementWithContent($type, $pageTitle, $elementTitl
         }
     }
 
+    /**
+     * @Given /a "([^"]+)" "([^"]+)" with a "([^"]+)" searchable fields block/
+     *
+     * @param string $pageTitle
+     * @param string $type
+     * @param string $elementTitle
+     */
+    public function thePageHasASearchableFieldsBlock($type, $pageTitle, $elementTitle)
+    {
+        $elementalArea = $this->getElementalArea($type, $pageTitle);
+        $elementalArea->Elements()->add(
+            $this->getFixtureFactory()->createObject(ElementalSearchableFieldsBlock::class, $elementTitle)
+        );
+    }
```

Comment:
This is unnecessarily narrow in scope. Just like you add the page _type_ (i.e. page class), you should add the block class in the behat test. That way we can reuse the same step for creating a page with any block class in it without having to make a new method for each block class.